django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -1045,16 +1045,23 @@ def test_template_used(self):
             self.assertEqual(response.status_code, 200)
             self.assertTemplateUsed(response, "base.html")
             self.assertTemplateUsed(response, "chant_create.html")
-
-    def test_next_chant_signal(self):
+    
+    def test_volpiano_signal(self):
         source = make_fake_source()
-        chant_1 = make_fake_chant(source=source, folio="001r", sequence_number=1)
-        response = self.client.post(
+        self.client.post(
+            reverse("chant-create", args=[source.id]), 
+            {"manuscript_full_text_std_spelling": "ut queant lactose", "folio": "001r", "sequence_number": "1", "volpiano": "9abcdefg)A-B1C2D3E4F5G67?. yiz"}
```

Comment:
Why this particular Volpiano?

Reply:
An excellent question! I annotated this in one of the commits I just pushed.